Info.plist

```diff
@@ -559,14 +549,17 @@
 			<string>LHA archive</string>
 			<key>UTTypeIconFile</key>
 			<string>lha</string>
+			<key>UTTypeIcons</key>
+			<dict>
+				<key>UTTypeIconBackgroundName</key>
+				<string>archive-fill</string>
+				<key>UTTypeIconText</key>
+				<string>lha</string>
+			</dict>
 			<key>UTTypeIdentifier</key>
 			<string>public.archive.lha</string>
 			<key>UTTypeTagSpecification</key>
 			<dict>
-				<key>com.apple.ostype</key>
-				<array>
-					<string>LHA </string>
-				</array>
```

Comment:
Is there a reason for this removal?

Reply:
Xcode wanted to. Support for file type codes has been removed in Snow Leopard.